### PR TITLE
[TW-458] Doc API definition to collection sync

### PR DESCRIPTION
**Pull request by mark-anthony-postman**

### CODE REVIEW

File: src/pages/docs/designing-and-developing-your-api/developing-an-api/adding-api-elements.md [hunk 1]
```diff
@@ -60,32 +59,58 @@ To add an existing collection to your API, do the following:
...
+## Keeping a collection in sync with an API
+
+When you [generate a collection](#generating-a-collection) from an API definition, Postman creates a collection with requests that match what's defined in the API. If you update the API definition, for example by adding a path, those changes won't be reflected in the generated collection.
+
+In this case, you could repeat the process of generating a collection from the API definition. However, this new collection wouldn't include any changes you might have made to the first collection you generated. For example, you might have added documentation content or endpoints for test setup that you want to keep. Instead, Postman can offer suggestions for updating the collection based on the changes to the API definition.
+
+### Enabling collection update suggestions
+
+You can enable update suggestions for any collection that's been added to an API. When suggestions are enabled, Postman will detect you when there are differences between the API definition and the collection and will offer suggestions for updating the collection.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/designing-and-developing-your-api/developing-an-api/adding-api-elements.md:86]:
```suggestion
You can enable update suggestions for any collection that's been added to an API. When suggestions are enabled, Postman will detect when there are differences between the API definition and the collection and will offer suggestions for updating the collection.
```

File: src/pages/docs/designing-and-developing-your-api/developing-an-api/adding-api-elements.md [hunk 2]
```diff
@@ -60,32 +59,58 @@ To add an existing collection to your API, do the following:
...
+
+> If you want to turn off suggestions, select the more actions icon <img alt="More actions icon" src="https://assets.postman.com/postman-docs/icon-more-actions-v9.jpg#icon" width="16px"> next to a collection and select **Disable update suggestions from definition**.
+
+### Updating a collection based on the API definition
+
+When suggestions are enabled, the update collection icon <img alt="Update collection icon" src="https://assets.postman.com/postman-docs/v10/icon-update-collection-v10.jpg#icon" width="16px"> appears next to a collection if Postman detects differences between the collection and the API definition. Differences can occur if the API definition is changed, for example by adding, removing, or modifying endpoints. Differences can also occur if requests are added, removed, or modified in the collection. You can review the differences and, if you choose, update the collection with the suggested changes.
+
+To update a collection based on the API definition, do the following:
+
+1. Select the update collection icon next to the collection on the API's overview.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/designing-and-developing-your-api/developing-an-api/adding-api-elements.md:101]:
```suggestion
1. Select the update collection icon <img alt="Update collection icon" src="https://assets.postman.com/postman-docs/v10/icon-update-collection-v10.jpg#icon" width="16px"> next to the collection on the API's overview.
```

**Review by MatthewPinsker-postman — APPROVED** [review]
LGTM!

**Review by jkonrath-postman — APPROVED** [review]
LGTM

File: src/pages/docs/designing-and-developing-your-api/developing-an-api/adding-api-elements.md [hunk 3]
```diff
@@ -60,32 +59,58 @@ To add an existing collection to your API, do the following:
...
+1. On the API's overview, select the more actions icon <img alt="More actions icon" src="https://assets.postman.com/postman-docs/icon-more-actions-v9.jpg#icon" width="16px"> next to a collection and select **Enable update suggestions from definition**.
+
+> If you want to turn off suggestions, select the more actions icon <img alt="More actions icon" src="https://assets.postman.com/postman-docs/icon-more-actions-v9.jpg#icon" width="16px"> next to a collection and select **Disable update suggestions from definition**.
+
+### Updating a collection based on the API definition
+
+When suggestions are enabled, the update collection icon <img alt="Update collection icon" src="https://assets.postman.com/postman-docs/v10/icon-update-collection-v10.jpg#icon" width="16px"> appears next to a collection if Postman detects differences between the collection and the API definition. Differences can occur if the API definition is changed, for example by adding, removing, or modifying endpoints. Differences can also occur if requests are added, removed, or modified in the collection. You can review the differences and, if you choose, update the collection with the suggested changes.
```
Comment (shashankawasthi88) [inline src/pages/docs/designing-and-developing-your-api/developing-an-api/adding-api-elements.md:97]: @mark-anthony-postman the update collection icon has been changed, can you fetch it from the latest build on Beta or preview?

**Review by shashankawasthi88 — COMMENTED** [review]
Looks good but icon needs to be updated

File: src/pages/docs/designing-and-developing-your-api/developing-an-api/adding-api-elements.md (hunk 3 above)
Comment (mark-anthony-postman) [reply src/pages/docs/designing-and-developing-your-api/developing-an-api/adding-api-elements.md:97]: @shashankawasthi88 Thanks for letting me know! I've updated the screenshots with the new icon

**Review by shashankawasthi88 — APPROVED** [review]
Looks good

**Pull request merged**